nsot/models.py

```diff
@@ -793,7 +793,7 @@ def get_next_network(self, prefix_length, num=None, as_objects=True):
         subnets = cidr.subnets(new_prefix=prefix_length)
 
         # Exclude children that are in busy states.
```

Comment:
It doesn't seem like this statement is accurate anymore. Especially given references below to busy children.